frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -123,25 +123,79 @@ const HostSummary = ({
   );
 
   const renderDiskSpace = () => {
+    const diskSpaceTooltip = () => {
+      const diskSpaceAvailable = titleData.gigs_disk_space_available;
+      switch (true) {
+        case diskSpaceAvailable < 16:
+          return (
+            <span className={`${baseClass}__tooltip-text`}>
+              Not enough disk space <br />
+              available to install most <br />
+              small operating systems <br />
+              updates.
+            </span>
+          );
+        case diskSpaceAvailable < 32:
+          return (
+            <span className={`${baseClass}__tooltip-text`}>
+              Not enough disk space <br />
+              available to install most <br />
+              large operating systems <br />
+              updates.
+            </span>
+          );
+        default:
+          return (
+            <span className={`${baseClass}__tooltip-text`}>
+              Enough disk space available <br />
+              to install most operating <br />
+              systems updates.
+            </span>
+          );
+      }
+    };
+
+    const diskSpaceIndicator = () => {
+      const diskSpaceAvailable = titleData.gigs_disk_space_available;
+      switch (true) {
+        case diskSpaceAvailable < 16:
+          return "red";
+        case diskSpaceAvailable < 32:
+          return "yellow";
+        default:
+          return "green";
+      }
+    };
```

Comment:
Interesting approach to a conditional switch statement. We typically use if / return patterns, but this is certainly readable. I googled it and turns out to be one of [those spaces vs. tabs debates](https://medium.com/trabe/using-switch-true-in-javascript-986e8ad8ae4f). To me it's readable and shows clear intent (along with your great function naming) so let's roll with it!